src/delivery-options/DeliveryOptions.vue

```diff
@@ -384,6 +394,21 @@ export default {
       };
     },
 
+    /**
+     * Determine whether the delivery date should be included.
+     *
+     * @param {Object} exportValues
+     * @returns {boolean}
+     */
+    shouldShowDeliveryDate(exportValues) {
+      const isPackage = PACKAGE_TYPE_PACKAGE === exportValues[PACKAGE_TYPE];
+      const isDomesticShipment = this.configBus.address.cc === (NL || BE);
```

Comment:
Dit werkt niet, dit wordt altijd NL. Wat je bedoelt is `[NL, BE].includes(cc)`, maar dan nog is dat niet wat "domestic" bij ons betekent.